tests/lang/core.pk

```diff
@@ -50,6 +59,17 @@ for i in 0..1000
   assert(abs(sin(i) / cos(i) - tan(i)) < threshold)
 end
 
+assert(abs(cosh(PI/2) - 2.5) < 0.01 )
+assert(abs(tanh(PI/4) - 1.0) < 0.5 )
+for i in 0..100
+  assert(abs(sinh(i) / cosh(i) - tanh(i)) <= 0.01)
+end
+
+assert(abs(acos(PI/4) - 1) < 0.35)
+assert(abs(atan(PI/4) - 0.5) < 0.2)
+for i in -1..1
+  assert(abs(asin(i) / acos(i) - atan(i)) <= 0.29)
```

Comment:
`asin(i) / acos(i)` isn't equal to `atan(i)`. use something like `sin(asin(x)) ~= x` (same of cos and tan). use the threshold for `~=` .

Reply:
A bit of help here @ThakeeNathees, this part is a little unclear. The use of `sin(asin(x)) ~= x` for example errors out with `Error: Expected statement end with '\n' or ';'.`. Are my missing something?

Reply:
No by `~=`, what I meant was, it's almost equal, so you have to use a threshold here like this.
```ruby
assert(abs(sin(asin(x)) - x) < threshold)
```

Adding `~=` as an operator to check almost equal number values (as opposed to `==`) might be something cool, but I've never seen any language is using that instead it a syntax error.

Reply:
Done!